RFS/src/main/java/org/opensearch/migrations/bulkload/lucene/version_9/IndexReader9.java

```diff
@@ -25,7 +25,7 @@ public LuceneDirectoryReader getReader() throws IOException {
             var commits = DirectoryReader.listCommits(directory);
             var latestCommit = commits.get(commits.size() - 1);
 
-            var reader = DirectoryReader.open(latestCommit);
+            var reader = DirectoryReader.open(latestCommit, 0, null);
```

Comment:
@AndreKurait this is the specific fix to support reading 6.8 with the backwards compatibility codecs.